src/drive/actions/index.js

```diff
@@ -84,49 +84,138 @@ export const openFolder = (folderId, client) => {
         cancelSelection: true
       }
     })
-    // TODO: temp fix for the thumbnails links on mobile
-    if (isCordova() && client) {
-      client.fetchFilesForLinks(folderId).then(resp =>
-        dispatch({
-          type: 'FETCH_FILES_LINKS_SUCCESS',
-          folderId,
-          files: resp
-        })
-      )
-    }
     try {
       const settings = getState().settings
-      const offline = settings.offline && settings.firstReplication
-      const folder = await cozy.client.files.statById(folderId, offline)
-      const parentId = folder.attributes.dir_id
-      const parent =
-        !!parentId &&
-        (await cozy.client.files.statById(parentId, offline).catch(ex => {
-          if (ex.status === 403) {
-            console.warn("User don't have access to parent folder")
-          } else {
-            throw ex
-          }
-        }))
-      const contents = folder.relationships.contents
-      // folder.relations('contents') returns null when the trash is empty
-      // the filter call is a temporary fix due to a cozy-client-js bug
-      const files =
-        folder.relations('contents').filter(f => f !== undefined) || []
+      const offline =
+        settings.offline && settings.firstReplication && settings.indexes
```

Comment:
Not  a big deal, but I find `firstReplication` confusing. It's not obvious if it means  `the first replication is in progress` or `the first replication has been done`.

Reply:
I agree, but this code was here for a long time, I didn't want to touch everything